src/util/format.ts

```diff
@@ -93,21 +94,30 @@ export function formatTpl(
         return '';
     }
 
-    const $vars = paramsList[0].$vars || [];
-    for (let i = 0; i < $vars.length; i++) {
-        const alias = TPL_VAR_ALIAS[i];
-        tpl = tpl.replace(wrapVar(alias), wrapVar(alias, 0));
+    const isTimeAxis = paramsList[0].axisType.indexOf('time') >= 0;
```

Comment:
@Ovilia `axisType` might be `undefined
![Screen Shot 2020-07-29 at 14 56 31](https://user-images.githubusercontent.com/20318608/88767091-b8e74b00-d1ab-11ea-98f6-a6ae657db1f6.png)
`

Reply:
For some charts which don't have `axis` like `tree` `treemap` `graph`

Reply:
Also found this issue.

Reply:
Fixed in https://github.com/apache/incubator-echarts/commit/353fee9d8d85be4727a95efbd021e82d1a77a060

Reply:
In the latest commit, I only saw the change of css style, is there anything forgotten commit?

Reply:
Unfortunately yes. Committed the missded file.